setup.py

```diff
@@ -4,15 +4,20 @@
 #
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
+import distutils.command.build
```

Comment:
All this new stuff in setup.py... Should some of it maybe be refactored into a separate module?